index.html

```diff
@@ -0,0 +1,99 @@
+<html>
+    <head>
+        <title>Guillain</title>
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" />
+        <link rel="stylesheet" href="style.css" />
+    </head>
+    <body class="container-fluid">
+        <div style="position: fixed" class="mt-4">
+            <nav id="navbar" class="nav-fill">
+                <header class="h3 text-secondary">Lorem Ipsum!</header>
+                <ul class="nav flex-column float-left">
+                    <li class="nav-item">
+                        <a href="#What_is_Lorem_Ipsum" class="nav-link">What is Lorem Ipsum?</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#Where_does_it_come_from" class="nav-link">Where does it come from?</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#Why_do_we_use_it" class="nav-link">Why do we use it?</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#Where_can_I_get_some" class="nav-link">Where can I get some?</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#standard_Lorem_Ipsum" class="nav-link">The standard Lorem Ipsum passage</a>
+                    </li>
+                </ul>
+            </nav>
+        </div>
+        <main class="mt-4" style="width: 78%; position: absolute; right: 0;" id="main-doc">
```

Comment:
Move all your CSS code in one file

Reply:
I think it would be best if you adopt a more polite and kind tone! for example:
> I kindly suggest you to keep all your CSS code in one file. `(because it is a project requirement, or any other reason you might have)`